Regression/Checksum/benchmarks_json/ElectrostaticSphereEB.json

```
@@ -0,0 +1,9 @@
+{
+  "lev=0": {
+    "Ex": 376867.5571515656,
+    "Ey": 376867.55715156574,
+    "Ez": 376867.5571515656,
+    "phi": 56106.09774654215,
+    "rho": 0.0
+  }
+}
```

```suggestion
}

```
Just to be consistent with style, there's no end of file newline here.